server/src/main/java/com/cloud/alert/AlertManagerImpl.java

```diff
@@ -718,6 +718,11 @@ public void sendAlert(AlertType alertType, long dataCenterId, Long podId, Long c
         mailProps.setContent(content);
         mailProps.setContentType("text/plain");
 
+        if (recipients == null) {
```

Comment:
cc @DaanHoogland @GutoVeronezi can you review this thanks.

Reply:
Nevermind Daan already LGTM'd. Let's wait for the test.

Reply:
Maybe we could do this validation before instantiating the properties.